### PR TITLE
v1 prefix added

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Pull Request

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [x] Bug Fix
- [x] Feature
- [x] Documentation Update
- [ ] Code style update (formatting, local variables)
- [ ] Breaking Change (fix or feature that would cause existing functionality to change)

## Linked tickets

- https://digicatapult.atlassian.net/browse/VR-112

## High level description

Make the API routes more consistent with other TSOA based projects

## Detailed description

Updated the routes to be more consistent with other TSOA projects. This included the v1 prefix to all routes, except the health endpoint, adding the health endpoint, updating where swagger can be found now on `/swagger` and where api docs can be found, now on `/api-docs`. 

## Describe alternatives you've considered

N/A

## Operational impact

Adheres to patterns followed in other projects.

## Additional context

N/A

### CODE REVIEW

File: README_VERITABLE.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ To get a minimal version of the agent running the following command is sufficien
 docker-compose up --build -d
 ```
 
-The agent is accessible via a `Swagger` interface (OpenAPI) on port `3000`.
+The agent is accessible via a `Swagger` interface (OpenAPI) on port `3000`. (The API docs are available at `http://localhost:3000/api-docs`.)
```
Comment (rmlearney-digicatapult) [inline README_VERITABLE.md:72]: No need for brackets around second sentence.

File: README_VERITABLE.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ The project aims to enable supply chains to share insights and data across multi
 
 ## Quick start
 
-The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI that is provided with the server. The docs can be viewed on the `/docs` endpoint (e.g. http://localhost:3000/docs).
+The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI that is provided with the server. The docs can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
```
Comment (mattdean-digicatapult) [inline README_VERITABLE.md:44]: kind of incorrect. The SwaggerUI is viewed at `/swagger` but the OpenAPI spec is server at `/api-docs`

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

**Review by mattdean-digicatapult — DISMISSED** [review]

**Review by n3op2 — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/veritable-cloudagent",
-  "version": "0.8.8",
+  "version": "1.0.0",
```
Comment (dblane-digicatapult) [inline package.json:3]: This should be 0.9.0 as its a pre v1 change.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (dblane-digicatapult) [reply package.json:3]: @mattdean-digicatapult Thoughts?

Comment (Ellenn-A) [reply package.json:3]: my ticket says bump major version ...

Comment (dblane-digicatapult) [reply package.json:3]: In Semver, prior to v1.0.0 a minor bump is considered a major version bump due to the rapidly changing APIs and structure.

https://semver.org/#doesnt-this-discourage-rapid-development-and-fast-iteration

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**